modules/text/FindCaffe.cmake

```diff
@@ -0,0 +1,14 @@
+# Caffe package for CNN Triplet training
+unset(Caffe_FOUND)
+
+find_path(Caffe_INCLUDE_DIR NAMES caffe/caffe.hpp caffe/common.hpp caffe/net.hpp caffe/proto/caffe.pb.h caffe/util/io.hpp caffe/vision_layers.hpp
+  HINTS
+  /usr/local/include)
```

Comment:
this assumes caffe is installed on the system directories. What if caffe is installed in an INSTALL_PREFIX that could be specified?